File: 221213/b_2002_junh.py

N = int(input())
dg = [input() for _ in range(N)]
ys = [input() for _ in range(N)]

dic = {dg[i]:i for i in range(N)}

answer = 0
for i in range(N):
    prv = dic[ys[i]]
    for j in range(i+1, N):
        if dic[ys[j]] < prv:
            answer += 1
            break

print(answer)

File: 221213/b_2002_junh.swift

let N = Int(readLine()!)!

var dg = [String]()
var ys = [String]()

for _ in 0 ..< N {
    dg.append(readLine()!)
}
for _ in 0 ..< N {
    ys.append(readLine()!)
}

var dic = [String:Int]()
for i in 0 ..< N {
    dic[dg[i]] = i
}
 
var answer = 0
for i in 0 ..< N {
    let org = dic[ys[i]]!
    for j in i+1 ..< N {
        if dic[ys[j]]! < org {
            answer += 1
            break
        }
    }
}
print(answer)

